package.json

```diff
@@ -30,7 +33,8 @@
     "test": "jest --coverage",
     "test:ui": "playwright test",
     "tsc": "tsc -p tsconfig.json",
-    "typecheck": "tsc --noEmit -p tsconfig.json"
+    "typecheck": "tsc --noEmit -p tsconfig.json",
+    "watch:buildto": "node watchBuild.js"
```

Comment:
Why don't we use the [watch-copy-build](https://github.com/terrestris/watch-build-copy) here?